src/sentry/south_migrations/0471_global_saved_search_types.py

```diff
@@ -0,0 +1,1377 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+
+class Migration(DataMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```

Comment:
@mattrobenolt Just going to let this one run automatically, we're basically updating <10 rows and adding one, so I think that's fine?